ui/src/hooks/useKeyboardShortcuts.ts

```diff
@@ -0,0 +1,20 @@
+import { useEffect } from 'react';
+import { useKeyPress } from 'reactflow';
+import { useUndoRedo } from '../singletons/store';
+
+export const useKeyboardShortcuts = () => {
```

Comment:
:question: Can we keep this file close to where it's used? So maybe in the same directory as `FlowCanvas.tsx`.